README.md

```diff
@@ -1,5 +1,69 @@
+
+There are sometimes we find ourselves in a need of recalling our sweet memories of what we talked with our friends in the past. Hence our mobile phones, or tablets don't have enough space to save the logs, and SMS for us for a long time like 1, 2, 3 or even 10 years.
+
+Here Phone History Store helps in that situation, you can store your sms, your call logs, and you phone book as well, for any desired time. It helps you manage and save securely all your phone history, and you can get your phone history wherever you are, even if you lost your device, but you history will remain with you.
+
+## Project description
+This project *Phone History Store Frontend* is the User Interface of [the backend project](https://github.com/nezago/phone-history-store-backend).
+
+It sync the calls, and SMS to a remote store, and whenever a user needs them, he/she can get them wherever he/she wants them, either on his telephone, or tablet, or computer.
+
+The main logic is that whenever you make, receive, or miss a new call this app will automaticall take the log and saves it on your local store and in a matter of time if you get connected to the internet, it will just send the new record to your remote store. 
+
+## Built with
+1. [React native](https://reactnative.dev/)
+1. [SASS](https://sass-lang.com/)
+1. [Java](https://www.java.com/en/)
+
+## Tools
+1. [VSCode](https://code.visualstudio.com/)
+1. [Git](https://git-scm.com/)
+1. [GitHub](https://www.github.com/)
+1. [Codacy](https://www.codacy.com/)
+1. [Codeclimate](https://codeclimate.com/)
+
+## Running and testing it locally
+To have a local copy up and running of this project just follow the following steps:
+
+### Prerequisites
+1. [NodeJS version 10 or higher](https://nodejs.org/en/)
+1. You need to have either Android or iPhone dev environment set on your local machine
+
+### Setup
+1. Clone this Repository by running `git clone https://github/nezago/phone-history-store-frontend.git`
+1. Go inside the cloned Repository by running `cd phone-history-store-frontend`
+1. Install all needed dependecies by running `npm install`
+
+### Usage
+1. Run `npm start` => To start local server
+1. Run `npm run android` => To run this app on android device or `npm run ios` to run this app on ios devices
+
+### Tests
+~~1. Run `npm test` => To run test suites on your local machine~~
+
+## Contributions
+This project is an open source project, this means anyone is allowed to contribute to it.
+
+There two ways of contributing to this project:
+
+1. If you see something wrong or not working, please check [the issue tracker section](https://github.com/nezago/phone-history-store-frontend/issues), if that problem you met is not in already opened issues then open the issue by click on issue button
+
+1. If you have a solution to that, follow the below steps to contribute:
+    1. Fork this repository
```

Comment:
Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, remove 1 space](https://app.codacy.com/gh/nezago/phone-history-store-frontend/pullRequest?prid=5868903)